compiler/rustc_interface/src/tests.rs

```diff
@@ -714,7 +714,7 @@ fn test_unstable_options_tracking_hash() {
     untracked!(unpretty, Some("expanded".to_string()));
     untracked!(unstable_options, true);
     untracked!(validate_mir, true);
-    untracked!(verbose, true);
+    untracked!(verbose_internals, true);
```

Comment:
Don't diagnostics influenced by this flag end up in the incr cache? It should actually be tracked I think.

Reply:
hmm, do we cache diagnostic *output*? even on errors? that seems very strange to me.

i don't mind changing it to tracked, i'm just confused why it's necessary.

Reply:
On errors we don't finalize the incr comp cache, but non-fatal diagnostics are cached afaik. Otherwise we would have to replay the query in question, which we may not be able to do if the query key is not reconstructible from the dep node fingerprint.